chatwoot/chatwoot
# fix: Conversation read update

Fixes https://github.com/chatwoot/chatwoot/issues/5688

**How to test?**
1. Create two conversations by using a web widget.
2. Minimize these two conversations.
3. Send an outgoing message from these two conversations (Messages sent by the agent are outgoing).
4. Open one of the conversations from the widget side.
5. Check the read status of the last messages. It should update only for the messages which the user reads.
